lms/static/scripts/frontend_apps/components/Dialog.js

```diff
@@ -135,7 +125,10 @@ Dialog.propTypes = {
    *
    * The "Cancel" button is added automatically if the `onCancel` prop is set.
    */
-  buttons: propTypes.arrayOf(propTypes.element),
+  buttons: propTypes.oneOfType([
```

Comment:
Note: This still causes an error until the node/element proeType polyfill" can be added

Reply:
That PR has landed, so this should now be resolved.